### PR TITLE
schedule monitoring first

### PR DESCRIPTION
**Pull request by philbrookes**
All components expect to be able to create monitoring resources, so it must be installed before all other components.

### CODE REVIEW

**Comment by david-martin** [comment]
@davidkirwan Can you take a look at this?

**Comment by mikenairn** [comment]
/hold

Lets ensure the e2e passes before merging

File: test/e2e/integreatly_test.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ var (
 	intlyNamespacePrefix             = "intly-"
 	installationName                 = "e2e-managed-installation"
 	bootstrapStage                   = "bootstrap"
+	monitoringStage				     = "monitoring"
```
Comment (mikenairn) [inline test/e2e/integreatly_test.go]: format?

Comment (philbrookes) [reply test/e2e/integreatly_test.go]: I did do a make code/fix, weird

**Comment by mikenairn** [comment]
@philbrookes You'll need to add an additional line here also https://github.com/integr8ly/integreatly-operator/blob/master/Makefile#L126

@pawelpaszki Will this affect the nightly/jenkins pipeline at all? Has that been modified to use the status checks for the Makefile or does it still do it it's own way?

**Comment by pawelpaszki** [comment]
@mikenairn nightly still uses the old version of install (till I will get another pipeline created specifically for OSD - likely this week)

**Comment by mikenairn** [comment]
/hold cancel

Tested a lot now and has past the e2e test 3 times in a row, no failures, looks good :+1:

**Pull request merged**